src/lib/components/Avatar/index.tsx

```diff
@@ -52,6 +54,10 @@ const AvatarComponent: FC<AvatarProps> = ({
             data-testid="flowbite-avatar-img"
             src={img}
           />
+        ) : placeholderInitials ? (
+          <div className="inline-flex overflow-hidden relative justify-center items-center w-10 h-10 bg-gray-100 rounded-full dark:bg-gray-600">
+            <span className="font-medium text-gray-600 dark:text-gray-300">{placeholderInitials}</span>
```

Comment:
Same thing here.

Reply:
@rluders addressed this in the latest commit